src/DevTools/__tests__/createMockNetworkLayer.test.ts

```diff
@@ -115,6 +115,58 @@ describe("createMockNetworkLayer", () => {
     }
   })
 
+  // TODO: upgrade graphql. The version we have does hardly any validaton of leaf values.
+  it.skip("complains with a helpful error when leaf field type is incorrect", async () => {
```

Comment:
Can you link to the updates in graphql-js that add this? (Assuming that that's what you mean.)